apps/logstream/src/main/java/com/datatorrent/apps/logstream/Application.java

```diff
@@ -19,18 +19,24 @@
 package com.datatorrent.apps.logstream;
 
 import java.net.URI;
-import java.util.*;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
 
 import org.apache.apex.malhar.contrib.misc.streamquery.SelectOperator;
 import org.apache.apex.malhar.contrib.misc.streamquery.condition.EqualValueCondition;
-
 import org.apache.commons.lang.StringUtils;
 import org.apache.hadoop.conf.Configuration;
 
+import com.datatorrent.api.DAG;
```

Comment:
why import changes?

Reply:
Got auto fixed according to checkstyle import order by IDE when making other changes in the file.

Reply:
Please change your IDE settings to not auto-format. Changes sets should not me mixed with formatting.